Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -282,8 +284,27 @@ void ABenchmarkGymGameMode::SpawnNPC(const FVector& SpawnLocation, const FBlackb
 	}
 }
 
+APlayerController* ABenchmarkGymGameMode::Login(UPlayer* NewPlayer, ENetRole InRemoteRole, const FString& Portal, const FString& Options, const FUniqueNetIdRepl& UniqueId, FString& ErrorMessage)
+{
+	// HACK: Spawn requests coming from NPC worker with invalid start point actors
+	SetPrioritizedPlayerStart(NULL);
+	return Super::Login(NewPlayer, InRemoteRole, Portal, Options, UniqueId, ErrorMessage);
+}
+
 AActor* ABenchmarkGymGameMode::FindPlayerStart_Implementation(AController* Player, const FString& IncomingName)
 {
+	if (!HasAuthority()) // HACK: Spawn requests coming from NPC worker with invalid start point actors
+	{
+		for (TActorIterator<APlayerStart> It = TActorIterator<APlayerStart>(GetWorld()); It; ++It)
```

Comment:
Can't you search for the PlayerStart of a specific type? Rather than all?

Reply:
Yeah although for that do we need a new class type? We could introduce a new one for this though (is this worth doing for a temp workaround?)